client/client-multi/client-objective-c/Kaa/avro/gen/EndpointGen.h

```diff
@@ -24,6 +24,7 @@
 #import "KAAUnion.h"
 
 
+
```

Comment:
Sorry, but it's not me :) 
I looked diff between the last pushed version of *recordObjC.h.vm* file and the new generated one and so i didn't make any changes that could affect on this. 

But yes, this line is redundant.